cfme/tests/automate/test_provisioning_dialogs.py

```diff
@@ -27,7 +27,7 @@ def test_provisioning_dialog_crud(appliance):
     assert dialog.exists
 
     with update(dialog):
-        dialog.diag_type = collection.HOST_PROVISION
+        dialog.diag_type = collection.VM_MIGRATE
```

Comment:
Question : Can you pls comment why is this change needed ? 
Host_provision is still an option for provisioning dialogs.

Reply:
It seems it's not anymore in CFME 5.10